addons/website_slides/static/src/tests/activity_tests.js

```diff
@@ -0,0 +1,121 @@
+odoo.define('website_slides/static/src/tests/activity_tests.js', function (require) {
+'use strict';
+
+const components = {
+    Activity: require('mail/static/src/components/activity/activity.js'),
+};
+
+const {
+    afterEach,
+    beforeEach,
+    start,
+} = require('mail/static/src/utils/test_utils.js');
+
+QUnit.module('website_slides', {}, function () {
+QUnit.module('activity_tests.js', {
+    beforeEach() {
+        beforeEach(this);
+
+        this.createActivityComponent = async activity => {
+            const ActivityComponent = components.Activity;
+            ActivityComponent.env = this.env;
+            this.component = new ActivityComponent(null, {
+                activityLocalId: activity.localId,
+            });
+            await this.component.mount(this.widget.el);
+        };
```

Comment:
Should use `mail/test_utils.js` helper function `createRootComponent`.

Reply:
Done